File: proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/FiscalMonthRangeException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class FiscalMonthRangeException extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = -5434654351452909152L;

	
	public FiscalMonthRangeException(int fiscalMonth) {
		super(String.format("Fiscal Year range is from 1 to 12. But it is %d.",
				fiscalMonth));
	}
}

File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/FiscalMonthRangeException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class FiscalMonthRangeException : Exception
    {
        public FiscalMonthRangeException(int fiscalMonth)
            : base(String.Format("Fiscal Year range is from 1 to 12. But it is %d.",
                fiscalMonth))
        {
        }
    }
}
